config/routes.rb

```diff
@@ -1,6 +1,9 @@
 Rails.application.routes.draw do
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts, only: [:index, :show, :create] do
+      resources :comments, only: [:create]
```

Comment:
- [ ] It would be better if we have a route for `GET /posts/:post_id/comments/new`

Reply:
@addod19 same thing you just want me to change the action name from create to new is that correct if not please explain in details

Reply:
> @addod19 same thing you just want me to change the action name from create to new is that correct if not please explain in details

Sorry for the misunderstanding, but I did not ask you to change. Let us add to it